src/anndata/_types.py

```diff
@@ -49,7 +49,8 @@
     H5Array,
     ZarrArray,
     ZappyArray,
-    BaseCompressedSparseDataset,
+    CSRDataset,
+    CSCDataset,
```

Comment:
This wasn't really correct to begin with since `BaseCompressedSparseDataset` isn't documented

Reply:
They look very sparsely documented though.

We should decide how much of that API we want to commit to and hide all attributes/methods we don’t want to support (especially that deprecated one)
 
![image](https://github.com/user-attachments/assets/c2e7e110-67d8-4fc9-841a-5a574ecc2eda)

Reply:
See https://github.com/scverse/anndata/pull/1608